### ISSUE TITLE
问下tensorflow的版本号，感谢！

### ISSUE THREAD

**Issue by LydiaLXH** [murray-z/FastText#1]
WARNING:tensorflow:From D:\Program\Python\FastText-master\fast_text.py:43: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See @{tf.nn.softmax_cross_entropy_with_logits_v2}.

**Comment by LydiaLXH** [comment]
> WARNING:tensorflow:From D:\Program\Python\FastText-master\fast_text.py:43: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
> Instructions for updating:
> 
> Future major versions of TensorFlow will allow gradients to flow
> into the labels input on backprop by default.
> 
> See @{tf.nn.softmax_cross_entropy_with_logits_v2}.
